templates/front/src/components/App.tsx

```diff
@@ -1,27 +1,21 @@
-import React, { useMemo } from 'react';
-import { createBrowserRouter, Outlet, RouterProvider } from 'react-router-dom';
-import { Logger } from 'simple-logging-system';
 import Layout from '@components/layout/Layout';
-import ErrorPage from '@components/pages/error/ErrorPage';
+import NotFoundPage from '@components/pages/error/NotFoundPage';
 import Home from '@components/pages/home/Home';
+import React from 'react';
+import { Logger } from 'simple-logging-system';
+import { ROUTE_HOME, UseRoute, useRoute } from '../router/Router';
 
 const logger: Logger = new Logger('App');
 
 export default function App() {
-  const router: ReturnType<typeof createBrowserRouter> = useMemo(() => createBrowserRouter([
-    {
-      path: '/',
-      element: <Layout><Outlet /></Layout>,
-      errorElement: <ErrorPage />,
-      children: [
-        {
-          index: true,
-          element: <Home />,
-        },
-      ],
-    },
-  ]), []);
+  const route: UseRoute = useRoute();
 
   logger.info('Render App');
-  return <RouterProvider router={router} />;
+
+  return (
+    <Layout>
+      {route.name === ROUTE_HOME && <Home />}
+      {route.name === false && <NotFoundPage />}
+    </Layout>
+  );
 }
```

Comment:
```suggestion
export default function App() {
  const route: UseRoute = useRoute();

  logger.info('Render App');

  if (route.name === false) {
    return (
      <Layout>
        <NotFoundPage />
      </Layout>
    );
  }
  
  return (
    <Layout>
      {
        [ROUTE_HOME]: (
          <DetailContratTermeGeneral contratTermeId={contratTermeId} />
        ),
      }[route.name]
    </Layout>
  );
}
```
j'aurai fait un truc comme ça pour la lisibilité